modules/mirror-registry-localhost.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-mirroring-installation-images.adoc
+
+[id="mirror-registry-localhost{context}"]
+
+= Local host mirroring with mirror registry for {product-title}
+
+This procedure introduces how to install Red Hat Quay on a local host using the `mirror-registry` tool. By doing so, users can create a registry to hold a mirror of {product-title} images.
+
+[NOTE]
+====
+Installing Red Hat Quay using the `mirror-registry` CLI tool makes several changes to your machine. Upon installation, an `/etc/quay-install` directory is created, which contains installation files, local storage, and the config bundle. Trusted SSH keys are generated in case the deployment target is the local host, and systemd files on the host machine are set up to ensure that container runtimes are persistent. Additionally, an initial user named `init` is created with an auto-generated password. All access credentials are printed at the end of the install routine.
+====
+
+.Procedure
+
+//placeholder link
+. Download the `mirror-registry.tar.gz` package for the latest version of mirror registry found on the link:https://github.com/quay/openshift-mirror-registry/releases[mirror registry releases page].
```

Comment:
QE was recommended to download  `mirror-registry.tar.gz` package from a image
```
podman cp $(podman create --rm registry-proxy.engineering.redhat.com/rh-osbs/openshift-mirror-registry-rhel8:v1.0-6'):/mirror-registry.tar.gz .
$ tar -xvzf mirror-registry.tar.gz
```
But here we recommand customer to download from github release page.
Which is the correct download method?

Reply:
Download page is preferred method for users.